dask/bytes/s3.py

```diff
@@ -13,15 +14,46 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_s3(storage_options=None):
```

Comment:
Thoughts on expanding storage_options and explicitly using keyword arguments like `username=`, `password=`, `key=`, `secret=`, etc..?